lexer.cpp

```diff
@@ -1,6 +1,6 @@
-#line 1 "lexer.cpp"
+#line 1 "/Users/swolchok/libgraphqlparser/lexer.cpp"
```

Comment:
Ugh, this keeps cropping up. cmake seems to run the tools by passing their absolute path. thanks.